src/components/nearMe/NearMeProfiles.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react'
+import { Box, Grid, ImageList, ImageListItem, Typography } from '@mui/material'
+import { LocationOn } from '@mui/icons-material'
+import { makeStyles } from 'tss-react/mui'
+import { getUsersNearMeData } from '../../actions/userServices'
+import { getColumns } from './helper'
+
+type isMobileProps = {
+  isMobile: boolean
+}
+type userNearMeObjectType = {
```

Comment:
Types with Capital letter, it's convention

Reply:
Changed TypesProps names to begin with capital letters.